panther_analysis_tool/rule.py

```diff
@@ -133,6 +139,18 @@ def __init__(self, config: Mapping):
         self.detection_id = config["id"]
         self.detection_version = config["versionId"]
 
+        # backwards compatible for legacy mocking
+        self._setup_exception = None
+        if "lastUpdatedVersion" in config:
+            try:
+                detection_panther_version = semver.VersionInfo.parse(config["lastUpdatedVersion"])
```

Comment:
is there a corresponding BE PR that populates this version field?

Reply:
Not currently, will be rewriting the BE PR as a lot of the code base has changed since I initially drafted that PR